File: CPP/Civitas.cpp
#include <iostream>
#include <string>
#include "Human.cpp"

using namespace std;

class Civitas : public Human {
    // attribute
    private:
        string asalUniversitas, emailEdu;

    // constructor
    public:
    Civitas() {
        this->asalUniversitas = "";
        this->emailEdu = "";
    } 

    Civitas(string asalUniversitas, string emailEdu) {
        this->asalUniversitas = asalUniversitas;
        this->emailEdu = emailEdu;
    }

    // getter and setter
    string getAsalUniversitas() {
        return this->asalUniversitas;
    }

    string getEmailEdu() {
        return this->emailEdu;
    }

    void setAsalUniversitas(string asalUniversitas) {
        this->asalUniversitas = asalUniversitas;
    }

    void setEmailEdu(string emailEdu) {
        this->emailEdu = emailEdu;
    }

    // method
    void absen() {
        cout << "bang sebut nama" << endl;
    }
};

File: PHYTON/Civitas.py
from Human import Human

class Civitas (Human):
    # kelas untuk menampung data civitas dan method absen

    # atribut
    __asalUniv = ""
    __emailEdu = ""

    # konstruktor
    def __init__(self):
        self.__asalUniv = ""
        self.__emailEdu = ""
    
    # setter getter
    def setAsalUniversitas(self, asalUniv):
        self.__asalUniv = asalUniv
    def setEmailEdu(self, emailEdu):
        self.__emailEdu = emailEdu

    def getAsalUniversitas(self):
        return self.__asalUniv
    def getEmailEdu(self):
        return self.__emailEdu
    
    # method
    def absen(self):
        print("bang sebut nama")
